### PR TITLE
Added an example to getSOAPRequest and getSOAPResponse

### PR DESCRIPTION
**Pull request by jordonr**
Provided a working example of using these to functions, and added the `webservice` parameter to `getSOAPRequest`.

### CODE REVIEW

File: data/en/getsoaprequest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name":"getSOAPRequest",
 	"type":"function",
-	"syntax":"getSOAPRequest()",
+	"syntax":"getSOAPRequest(webservice)",
```
Comment (pfreitag) [inline data/en/getsoaprequest.json]: Good catch here, this is good, however we also need to add it to the `params` array, like this:
```
"params": [
		{"name":"webservice","description":"The description.","required":false,"default":"","type":"string","values":[]}
],

Note: I didn't fill in a `description` above
```

Comment (jordonr) [reply data/en/getsoaprequest.json]: Should I also change the type from "string" to "any" as it's listed in the Lucee docs?

Comment (pfreitag) [reply data/en/getsoaprequest.json]: yes, should be `any` - good catch

File: data/en/getsoaprequest.json [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 	"returns":"any",
 	"related":["getSOAPRequestHeader","getSOAPResponse","getSOAPResponseHeader"],
 	"description":" Returns an XML object that contains the entire SOAP\n request. Usually called from within a web service CFC.",
-	"params": [],
+	"params": [
+		{"name":"webservice","description":"A webservice object as returned from the cfobject tag or the CreateObject function.","required":true,"default":"","type":"any","values":[]}
```
Comment (pfreitag) [inline data/en/getsoaprequest.json]: I don't think this argument `required` according to adobe docs it is optional

Comment (jordonr) [reply data/en/getsoaprequest.json]: I can update, I see that Lucee and the Adobe docs have that.  Also I tried to confirm that by using trycf.com and the following code fails on Lucee and ColdFusion 2018 and ColdFusion 2016:
```
soapURL = "https://graphical.weather.gov/xml/SOAP_server/ndfdXMLserver.php?wsdl";
ws = CreateObject("webservice", soapURL);
zipLatLong = ws.LatLonListZipCode("10001");
//res = getSOAPRequest(ws);
res = getSOAPRequest();
writeDump(res);
```

**Comment by jordonr** [comment]
Hello, just following up if there is anything I missed or need to do still.  Thanks!

**Comment by pfreitag** [comment]
Thanks @jordonr 

BTW if you want to do another PR you can change it to `type="any"` on that argument

**Pull request merged**